src/render/SDL_render.c

```diff
@@ -967,6 +1001,14 @@ SDL_Renderer *SDL_CreateRenderer(SDL_Window *window, int index, Uint32 flags)
     Android_ActivityMutex_Lock_Running();
 #endif
 
+/*CPU acceleration runs faster than integrated graphichs on Loongson at pressent.*/
+#if defined(__linux__) && defined(__loongarch__)
+    if (SDL_CheckIntegratedGraphics()) {
+        SDL_SetHint(SDL_HINT_FRAMEBUFFER_ACCELERATION, "0");
+        flags = SDL_RENDERER_SOFTWARE;
+    }
+#endif
```

Comment:
This will effectively make it impossible to create an opengl `SDL_Renderer` on those platforms. Is this ok for compatibility with non-SDL renderer using OpenGL?

Reply:
With `non-SDL renderer using OpenGL`, I mean a user project using OpenGL calls for drawing (including 3D) instead of using the SDL renderer.

Reply:
Thank you for your quick reply, it seems this change will not affect that behavior.